src/layouts/default.vue

```diff
@@ -1,17 +1,16 @@
 <template>
-  <div class="app grid h-screen overflow-hidden relative">
-    <div>
+  <div class="app grid relative">
+    <div class="sticky top-0 block z-40">
       <VTeleportTarget name="skip-to-content" :force-destroy="true" />
       <VMigrationNotice v-show="isReferredFromCc" />
       <VTranslationStatusBanner />
       <VHeader />
     </div>
-    <main
-      class="main embedded overflow-x-hidden"
-      :class="{ 'has-sidebar': isSidebarVisible }"
-    >
-      <Nuxt ref="mainContentRef" class="min-w-0 main-page" />
-      <VSidebarTarget class="sidebar" />
+    <main class="main embedded" :class="{ 'has-sidebar': isSidebarVisible }">
+      <Nuxt class="min-w-0 main-page" />
+      <VSidebarTarget
+        class="sidebar fixed pb-20 end-0 bg-dark-charcoal-06 border-s border-dark-charcoal-20 overflow-y-auto"
```

Comment:
The border color should be conditional here, something like:
`:class={ 'border-dark-charcoal-20': isSidebarVisible }`. Otherwise, you can see a gray border even when the sidebar is closed.

Reply:
Done!